src/cli/commands/public-cmds/init-cmd.ts

```diff
@@ -16,9 +16,9 @@ import { CommandOptions, LegacyCommand } from '../../legacy-command';
 export default class Init implements LegacyCommand {
   name = 'init [path]';
   skipWorkspace = true;
-  shortDescription = 'create or reinitialize an empty Bit scope or reinitialize an existing one';
+  shortDescription = 'create or reinitialize an empty workspace';
```

Comment:
this used also to init a scope (with the bare flag)

Reply:
the description should describe the main functionality and should not include its various options. 
'bit init --bare' is just one way of initializing a workspace (bare of the working directory)